assets/js/index.js

```diff
@@ -1,9 +1,9 @@
 function initialiseMap() {
-  let cowinMap = new CowinMap();
+  let c = new CowinMap();
```

Comment:
why not keep the variable name `cowinMap`?

Reply:
Will rever this change. I was testing something when this one seeped through.